www/data/docs_sidebar.yml

```diff
@@ -1,122 +1,195 @@
-      link: "/docs/concepts-supervisor/"
+      link: "#glossary-supervisor"
     - title: Plans
-      link: "/docs/concepts-plans/"
+      link: "#glossary-plan"
+    - title: Hooks
+      link: "#glossary-hooks"
     - title: Scaffolding
-      link: "/docs/concepts-scaffolding/"
+      link: "#glossary-scaffolding"
     - title: Packages
-      link: "/docs/concepts-packages/"
+      link: "#glossary-packages"
     - title: Keys
-      link: "/docs/concepts-keys/"
+      link: "#glossary-keys"
     - title: Studio
-      link: "/docs/concepts-studio/"
+      link: "#glossary-studio"
     - title: Services
-      link: "/docs/concepts-services/"
-    - title: Depot
-      link: "/docs/concepts-depot/"
-- title: Using Habitat
-  links:
-  - title: Create packages <span>&plus;</span>
-    link: "/docs/create-packages-overview/"
-    sub_links:
-    - title: Create plans
-      link: "/docs/create-plans/"
-    - title: Add configuration
-      link: "/docs/create-packages-configure/"
-    - title: Build packages
-      link: "/docs/create-packages-build/"
-    - title: Debug plans
-      link: "/docs/create-packages-debugging/"
-    - title: Create binary-only packages
-      link: "/docs/create-packages-binary-only"
-  - title: Run packages <span>&plus;</span>
-    link: "/docs/run-packages-overview/"
-    sub_links:
-    - title: Service groups
-      link: "/docs/run-packages-service-groups/"
-    - title: Topologies
-      link: "/docs/run-packages-topologies/"
-    - title: Configuration updates
-      link: "/docs/run-packages-apply-config-updates/"
-    - title: Upload files
-      link: "/docs/run-packages-upload-files/"
-    - title: Security
-      link: "/docs/run-packages-security/"
-    - title: Binding
-      link: "/docs/run-packages-binding/"
-    - title: Update strategy
-      link: "/docs/run-packages-update-strategy/"
-    - title: Multiple Services
-      link: "/docs/run-packages-multiple-services/"
-    - title: Export packages
-      link: "/docs/run-packages-export/"
-    - title: Monitoring
-      link: "/docs/run-packages-monitoring/"
-  - title: Share packages
-    link: "/docs/share-packages-overview/"
-  - title: Continuous deployment
-    link: "/docs/continuous-deployment-overview/"
-  - title: Container orchestration
-    link: "/docs/container-orchestration/"
-    sub_links:
-    - title: Amazon EC2 Container Service
-      link: "/docs/container-orchestration-ecs/"
-    - title: Apache Mesos and DC/OS
-      link: "/docs/container-orchestration-mesos/"
-    - title: Kubernetes
-      link: "/docs/container-orchestration-kubernetes/"
-  - title: Habitat internals <span>&plus;</span>
-    link: "/docs/internals-overview/"
+      link: "#glossary-services"
+    - title: Builder
+      link: "#glossary-builder"
+  - title: Internals <span>&plus;</span>
+    link: "/docs/internals"
     sub_links:
     - title: Supervisor
-      link: "/docs/internals-supervisor/"
+      link: "#supervisor-internals"
```

Comment:
Instead of internals, how about "Advanced Developer Documentation"

Reply:
I dunno, I think internals is a pretty standard fare title for the types of content in this section, but I could be convinced otherwise. Im pretty ambivalent so if you have a strong feeling i'll switch it.